internal/provider/resources/resource_deployment.go

```diff
@@ -850,25 +876,25 @@ func validateHostedConfig(ctx context.Context, data *models.DeploymentResource)
 			"Please provide is_development_mode",
 		)
 	}
-	if data.ResourceQuotaCpu.IsNull() {
+	if data.ResourceQuotaCpu.IsNull() && data.RemoteExecution.IsNull() {
```

Comment:
why do we need all these additional remote execution null checks?

Reply:
Because they cannot be set for Remote Execution Deployments.